src/kayenta/report/detail/reportException.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import { ICanaryState } from 'kayenta/reducers';
+import { ICanaryExecutionException } from 'kayenta/domain';
+import SourceLinks from './sourceLinks';
+
+import './reportExplanation.less';
+
+interface IReportExceptionProps {
+  exception: ICanaryExecutionException;
+}
+
+const ReportException = ({ exception }: IReportExceptionProps) => {
+  const message =
+    exception.details?.error ??
+    exception.details.errors?.join('\n') ??
```

Comment:
On the backend, I just used the error field because that's the only one I saw populated. Don't know if that's right or not. :-) 

I guess we could check on the backend to see what's going on there -- @csanden Do you know if both of these fields are present?

Reply:
Is there a case where details can be null here? If so, wouldn't it go like this:
exception.details?.error returns null because details is null/undefined so it executes the next line
but details is null so it will try to access a property on undefined.
My javascript is a little rusty so I may be missing some detail here, but won't that cause an error?

Reply:
@dwest-netflix you're right — this should be guarding against `details` being null in both cases. I think this code is a little too terse. I am going to refactor it a little to make it more clear what's happening.

@haleyw these error messages come from [Orca](https://github.com/spinnaker/orca/blob/71c688ef06ade1f01d58385ff4da02c3d334db05/orca-core/src/main/java/com/netflix/spinnaker/orca/exceptions/ExceptionHandler.java#L92), but it's hard to know which fields will be populated.